test/helpers/kubectl.go

```diff
@@ -420,9 +437,81 @@ func (kub *Kubectl) CiliumReport(namespace string, pod string, commands []string
 		fmt.Fprintln(wr, out.CombineOutput())
 	}
 	fmt.Fprint(wr, "StackTrace Ends\n")
+	kub.CiliumReportFull(namespace, pod)
 	return nil
 }
 
+//CiliumReportFull writes to testResultsPath all the output of a variety of
+//cilium commands to know the state of cilium
+func (kub *Kubectl) CiliumReportFull(namespace string, pod string) {
```

Comment:
I'm not a fan of this function name. What about `CiliumReportDump`? A dump is a widely used term, while "Full" is somewhat murky.

Reply:
Please capitalize Cilium in documentation.

Reply:
Incorrect grammar; "...dumps the output of a variety of Cilium commands to files in directory testResultsPath"?